File: packages/backend/functions/python/websocket/connect.py
import json


def handler(event, context):
    connection_id = event["requestContext"]["connectionId"]
    print(f"Client connected: {connection_id}")

    # Store connectionId in DynamoDB or your preferred storage

    return {
        "statusCode": 200,
        "body": json.dumps({"message": "Connected"}),
    }

File: packages/backend/functions/ts/websocket/connect.ts
import type { APIGatewayProxyHandler } from "aws-lambda";
import { success } from "../../../lib/response.js";

export const handler: APIGatewayProxyHandler = async (event) => {
  const connectionId = event.requestContext.connectionId;
  console.log(`Client connected: ${connectionId}`);

  // Store connectionId in DynamoDB or your preferred storage

  return success({ message: "Connected" });
};
